src/main/scala/MakeReleaseNotes.scala

```diff
@@ -109,7 +109,7 @@ object MakeReleaseNotes {
       </body>
     </html>"""
       case MarkDown => s"""---
-layout: news
+layout: inner-page-no-masthead
```

Comment:
/cc @szeiger you'll need this change when generating the 2.13.0-M2 release notes